docs/providers/aws/guide/serverless.yml.md

```diff
@@ -54,6 +54,7 @@ provider:
     skipPolicySetup: false # Prevents creation of default bucket policy when framework creates the deployment bucket. Default is false
     name: com.serverless.${self:provider.region}.deploys # Deployment bucket name. Default is generated by the framework
     maxPreviousDeploymentArtifacts: 5 # On every deployment the framework prunes the bucket to remove artifacts older than this limit. The default is 5
+    versioning: true # enable bucket versioning. Default is false
```

Comment:
let's specify here `false` as it's the default

Reply:
Updated